rcl/CMakeLists.txt

```diff
@@ -3,14 +3,68 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rcl)
 
 find_package(ament_cmake REQUIRED)
+find_package(rcl_interfaces REQUIRED)
+find_package(rmw REQUIRED)
+find_package(rmw_implementation REQUIRED)
+find_package(rmw_implementation_cmake REQUIRED)
+find_package(rosidl_generator_c REQUIRED)
 
+include_directories(include)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
```

Comment:
No, it causes issues on GCC 4.7 and it is not necessary to define that in order to use `stdatomic.h`, which is the only thing from "C11" that I use. Also Windows doesn't support it and it doesn't appear that it will.